angular/coding-styles/README.md

```diff
@@ -0,0 +1,157 @@
+# Angular Style Guide (Optional)
+
+<br>
+<br>
+
+**this page is in draft mode. it is optional to follow guidelines on this page.**
+
+<br>
+<br>
+
+## Style Guide
+
+### Base Style Guide
+
+The main programing language in Angular framework included with Typescript, HTML, SCSS. We follow
+
+- [Google style guide for HTML/CSS](https://google.github.io/styleguide/htmlcssguide.html), [Typescript](https://google.github.io/styleguide/tsguide.html)
+- [Kitty Giraudel style guide for SCSS](https://sass-guidelin.es/)
+- [Angular Official for Angular](https://angular.io/guide/styleguide).
+
+The rest of the document describes additions and clarifications below.
+
+### Typescript
+
+#### Indentation
+
+The indentation in typescript we use 2 spaces for every new line. we don't tabs or mix tabs.
```

Comment:
```suggestion
For the indentation in TypeScript, we use **2 spaces** for every new line. we don't use tabs or spaces mixed with tabs.
```

What do you think krub?

Reply:
Changes:
... for every new line. Do not use tabs or spaces mixed with tabs.